presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1348,7 +1372,7 @@ private Session withUnsupportedType(UnsupportedTypeHandling unsupportedTypeHandl
                 // PostgreSQL does not store zone, only the point in time
                 zonedDateTime -> DateTimeFormatter.ofPattern("'TIMESTAMP WITH TIME ZONE '''yyyy-MM-dd HH:mm:ss.SSS VV''").format(zonedDateTime.withZoneSameInstant(UTC)),
                 DateTimeFormatter.ofPattern("'TIMESTAMP '''yyyy-MM-dd HH:mm:ss.SSS VV''")::format,
-                zonedDateTime -> zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")));
+                zonedDateTime -> zonedDateTime.truncatedTo(MILLIS).withZoneSameInstant(ZoneId.of("UTC")));
```

Comment:
why truncate here?

and why only millis, where you're adding precision up to micro?

(if this is needed, a code comment is due)

Reply:
no longer valid